### PR TITLE
feat(page-header-standardised): Adding component - FRONT-1802

### PR DESCRIPTION
**Pull request by papegaill**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)
- [ ] The redirects in `src/website/src/routes/Redirects.jsx` are up-to-date

### CODE REVIEW

File: src/implementations/twig/components/page-header-standardised/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+- **"title"** (string) (default: '') Title of header
+- **"description"** (string) (default: '') Description of header
+- **"meta"** (string) (default: '') Meta of header
+- **"breadcrumb"** (associative array) (default: '') Predefined structure compatible with EC Breadcrumb
+- **"extra_classes"** (optional) (string) (default: '') Extra classes (space separated)
+- **"extra_attributes"** (optional) (array) (default: []) Extra attributes
+  - "name" (string) Attribute name, eg. 'data-test'
+  - "value" (string) Attribute value, eg: 'data-test-1'
+- **"\_compliance**"\_ (boolean) (default: false) Activates debug
```
Comment (planctus) [inline src/implementations/twig/components/page-header-standardised/README.md]:
```suggestion
- **"\_compliance\_**" (boolean) (default: false) Activates debug
```

File: src/implementations/twig/components/page-header-standardised/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+```shell
+npm install --save @ecl/twig-component-page-header-standardised
+```
+
+### Parameters
+
+- **"title"** (string) (default: '') Title of header
+- **"description"** (string) (default: '') Description of header
+- **"meta"** (string) (default: '') Meta of header
+- **"breadcrumb"** (associative array) (default: '') Predefined structure compatible with EC Breadcrumb
```
Comment (planctus) [inline src/implementations/twig/components/page-header-standardised/README.md]:
```suggestion
- **"breadcrumb"** (associative array) (default: '') Predefined structure for the ECL Breadcrumb
```

**Review by planctus — APPROVED** [review]
Just two very small things in the README file, the component looks fine to me.

**Review by planctus — APPROVED** [review]

**Comment by planctus** [comment]
@papegaill it seems we miss the dependency in the preset-dev for the vanilla component you added ;)

**Comment by planctus** [comment]
And there is some issues with the linter, @papegaill

File: src/website/src/pages/ec/standardised-template/page-header/docs/code.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  pageHeaderTitle,
+  pageHeaderMetaTitle,
+  pageHeaderMetaTitleDescription,
+} from '../demo';
+
+## Title
+
+<Playground
+  system="ec"
+  selectedKind="components-page-headers-page-header-standardised"
```
Comment (planctus) [inline src/website/src/pages/ec/standardised-template/page-header/docs/code.mdx]: i noticed this only when reviewing the page header core, i think it's better to rename the story and have the path to it like this: components-page-headers-standardised, like in v2

**Review by planctus — CHANGES_REQUESTED** [review]
left and additional comment inline, sorry, i've noticed it only now..

File: src/implementations/twig/components/page-header-standardised/page-header-standardised.html.twig [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  {% if _breadcrumb is not empty %}
+    {% include '@ecl/breadcrumb-standardised/breadcrumb-standardised.html.twig' with _breadcrumb|merge({
+      extra_classes: 'ecl-page-header-standardised__breadcrumb',
+      _compliance_: _compliance,
+      _compliance_inner_check_: true
+    }) only %}
+  {% endif %}
+  {% if _meta is not empty %}
+    <div class="ecl-page-header-standardised__meta">
+      <span class="ecl-u-type-uppercase">
```
Comment (planctus) [inline src/implementations/twig/components/page-header-standardised/page-header-standardised.html.twig]: So....we just released in ECL v2 an updated version of the css for these components, where the meta is uppercased in the CSS and i see that we have it in the css you are adding in the vanilla component, so this <span> can now disappear, i write this here but you know it's the same in all the families, sorry, i could only notice this now that i worked on aligning the twig component ;)

**Review by planctus — CHANGES_REQUESTED** [review]
It's just the span that we can remove from all the page-headers, thanks @papegaill

File: src/implementations/twig/components/page-header-standardised/page-header-standardised.html.twig (hunk 4 above)
Comment (papegaill) [reply src/implementations/twig/components/page-header-standardised/page-header-standardised.html.twig]: Nice catch! Updated on 3 page-headers ;)

**Comment by planctus** [comment]
We need to regenerate the snapshots @papegaill .. ;)

**Review by planctus — APPROVED** [review]

**Pull request merged**